### DISCUSSION TITLE
How do I get a screenshot from `scrapeUrl` when using the NodeJS SDK?

### DISCUSSION THREAD

**Discussion by OliverWales** [firecrawl/firecrawl#583] [Q&A] [answered]
According to https://docs.firecrawl.dev/v1-welcome#scrape-formats, screenshot is one of the available formats. However, when I try the following request I just get back the same result as if I had not set this parameter. The NodeJS types are not helpful as the params are typed as `[key: string]: any`.

```TS
result = await app.scrapeUrl(url, {
  formats: ['markdown', 'screenshot'],
})
```

How can I get a screenshot from the NodeJS SDK `scrapeUrl` function?

**Comment by nickscamara** [comment, marked as answer]
Hey Oliver, you should get it from `result.screenshot`. Looping in @rafaelsideguide here to take a look at the type issues.

**Reply by nickscamara** [reply]
Also, could you share the url you are trying? That can help us debug it.

**Reply by OliverWales** [reply]
Seems okay now, maybe I mistyped something or misread the response. Better types would help I think!

**Reply by ifx-querido** [reply]
and what's the solution for the python SDK? because when I run result.screenshot, I get the following error:  'dict' object has no attribute 'screenshot'

**Reply by rafaelsideguide** [reply]
@ifx-querido you need to access the value using bracket notation like this: `result['screenshot']`, since result is a dictionary.

```python
from firecrawl.firecrawl import FirecrawlApp
app = FirecrawlApp(api_key="fc-*")

result = app.scrape_url('docs.firecrawl.dev', {
  "formats": ['markdown', 'screenshot'],
})

print(result['screenshot'])
```

**Reply by ifx-querido** [reply]
Thank you Rafael, this worked perfectly. However, I'm now having issues with the full page screenshot. According to the scrape format documentation linked in the original post, and following the dictionary logic, I ran the following:

```
page_content = app.scrape_url(website, {
  "formats": ['markdown', 'screenshot', 'screenshot@fullPage'],
})

print(page_content['screenshot@fullPage'])
```
Here's the error message:
`KeyError: 'screenshot@fullPage'`
